src/ring_theory/coprime.lean

```diff
@@ -258,6 +283,32 @@ lemma mul_add_left_right_iff {x y z : R} : is_coprime x (x * z + y) ↔ is_copri
 lemma mul_add_right_right_iff {x y z : R} : is_coprime x (z * x + y) ↔ is_coprime x y :=
 ⟨of_mul_add_right_right, λ h, h.mul_add_right_right z⟩
 
+lemma neg_left {x y : R} (h : is_coprime x y) : is_coprime (-x) y :=
+begin
+  obtain ⟨a, b, h⟩ := h,
+  use [-a, b],
+  rwa neg_mul_neg,
+end
+
+lemma neg_left_iff (x y : R) : is_coprime (-x) y ↔ is_coprime x y :=
+⟨λ h, (neg_neg x) ▸ h.neg_left, is_coprime.neg_left⟩
+
+lemma neg_right {x y : R} (h : is_coprime x y) : is_coprime x (-y) :=
+begin
```

Comment:
is it possible to state this directly in terms of `neg_left` or does that require `neg_neg`? otherwise, looks good

Reply:
Yes, you can use symmetry:
```
lemma neg_right {x y : R} (h : is_coprime x y) : is_coprime x (-y) :=
h.symm.neg_left.symm
```